### ISSUE TITLE
The rule require-description-complete-sentence, is triggering incorrectly

### ISSUE THREAD

**Issue by khrome83** [gajus/eslint-plugin-jsdoc#7]
The following JSDoc comments trigger the rule `require-description-complete-sentence`, yet they appear to be valid.

``` javascript
/**
 * Impress Class, used to intialize a new Impress parser.
 * 
 * @class
 * @classdesc Impress Parser, and exposed API for node module
 * @param  {Object} [options={}] - Contains all options user can override
 * @param  {any} [reporter] - Reporter plugin to use to see progress
 * @param  {Array.<string>} [plugins=['defaults']] - Plugins to be enabled
 * @example
 * // Setups up a instance of Impress with default plugins
 * var Impress = require(impress),
 *     impress = new Impress();
 * @example
 * // Setups of a instance of Impress, modifying the plugins parsed
 * var Impress = require(impress),
 *     impress = new Impress({}, ['test', 'list']);
 */
```

``` javascript
/**
 * Runs HTML string through the impress parser to generate TreeAdapter to run all plugins enabled against it.
 * 
 * @param  {string} html - HTML markup to parse, either whole document or fragment
 * @param  {Object} [data={}] - Initial data to prime parser with
 * @returns {string} - HTML markup as string, removing any Impress commands
 */
```

**Comment by khrome83** [comment]
Console Output
``` npm
C:\Users\zanem\Documents\GitHub\impressjs\lib\index.js
  14:1  warning  Description must start with an uppercase character  jsdoc/require-description-complete-sentence
  41:1  warning  Description must start with an uppercase character  jsdoc/require-description-complete-sentence
```

**Comment by gajus** [comment]
I have fixed the:

> 14:1  warning  Description must start with an uppercase character  jsdoc/require-description-complete-sentence
> 41:1  warning  Description must start with an uppercase character  jsdoc/require-description-complete-sentence

The reason for the error was the leading dash.

Given the above example, you will still get errors saying that description/sentence must end with a period. Thats expected behaviour though.

**Comment by khrome83** [comment]
Awesome, I will update later tonight and confirm. Lots of work put into the 2.x release in such a short amount of time. Great job @gajus
